pkg/ovsdb/ovsdb.go

```diff
@@ -58,10 +88,18 @@ func NewOvsBridgeDriver(bridgeName, socketFile string) (*OvsBridgeDriver, error)
 	ovsDriver := new(OvsBridgeDriver)
 
 	if socketFile == "" {
-		socketFile = "/var/run/openvswitch/db.sock"
+		socketFile = "unix:/var/run/openvswitch/db.sock"
 	}
 
-	ovsDB, err := libovsdb.ConnectWithUnixSocket(socketFile)
+	dbmodel, err := client.NewDBModel("Open_vSwitch",
```

Comment:
This is the same as the model from NewOvsDriver, no? Please create a helper function that would initialize the model and call it from both functions

Reply:
Ok. I made a helper function.